extensions/amp-smartlinks/0.1/test/validator-amp-smartlinks.html

```diff
@@ -0,0 +1,37 @@
+<!--
+	Copyright 2018 The AMP HTML Authors. All Rights Reserved.
+	Licensed under the Apache License, Version 2.0 (the "License");
+	you may not use this file except in compliance with the License.
+	You may obtain a copy of the License at
+			http://www.apache.org/licenses/LICENSE-2.0
+	Unless required by applicable law or agreed to in writing, software
+	distributed under the License is distributed on an "AS-IS" BASIS,
+	WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+	See the License for the specific language governing permissions and
+	limitations under the license.
+-->
+<!--
+	Test Description:
+	Valid amp-smartlinks tag
+-->
+<!doctype html>
+<html ⚡>
+<head>
+	<meta charset="utf-8">
+	<link rel="canonical" href="./regular-html-version.html">
+	<meta name="viewport" content="width=device-width,minimum-scale=1">
+	<style amp-boilerplate>body{-webkit-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-moz-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-ms-animation:-amp-start 8s steps(1,end) 0s 1 normal both;animation:-amp-start 8s steps(1,end) 0s 1 normal both}@-webkit-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-moz-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-ms-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-o-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}</style><noscript><style amp-boilerplate>body{-webkit-animation:none;-moz-animation:none;-ms-animation:none;animation:none}</style></noscript>
+	<script async custom-element="amp-smartlinks" src="https://cdn.ampproject.org/v0/amp-smartlinks-0.1.js"></script>
+	<script async src="https://cdn.ampproject.org/v0.js"></script>
+</head>
+<body>
+    <amp-smartlinks
```

Comment:
We tend to prefix each test "case" with whether it is intended to be valid or invalid. When invalid we state why it's invalid.

Before this line, add `    <!-- Valid -->`

Example of invalid test:
```
    <! Invalid: missing mandatory attribute nrtv-account-name -->
    <amp-smartlinks>
        layout="nodisplay"
        linkmate="true"
        exclusive-links="false"
        link-attribute="href"
        link-selector="a">
    </amp-smartlinks>
```